cypress/component/rich-text/RichTextEditor.Marks.spec.ts

```diff
@@ -21,7 +21,12 @@ describe('Rich Text Editor - Marks', { viewportHeight: 2000, viewportWidth: 1000
   });
 
   const findMarkViaToolbar = (mark: string) => {
-    if (mark === 'code' || mark === 'superscript' || mark === 'subscript') {
+    if (
+      mark === 'code' ||
```

Comment:
what do you think about?
`['code', 'superscript', 'subscript', 'strikethrough'].includes(mark)`

you could also extract it into a constant outside of the function and re-use it in `toggleMarkViaToolbar`